src/NServiceBus.Core/obsoletes.cs

```diff
@@ -973,6 +973,8 @@ public void AddSatelliteReceiver(string name, string transportAddress, Transport
         {
             throw new NotImplementedException();
         }
+
+        const string AddSatelliteOverloadMemberDefinition = "AddSatelliteReceiver(string name, string transportAddress, PushRuntimeSettings runtimeSettings, Func<RecoverabilityConfig, ErrorContext, RecoverabilityAction> recoverabilityPolicy, Func<IBuilder, MessageContext, Task> onMessage)";
```

Comment:
why adding the const here? If this is only used here https://github.com/Particular/NServiceBus/pull/5011/files#diff-287c0fdac06c80a87ce22cb7ee81ac17R971, why not inline it?

Reply:
👍